install/installation-archlinux.md

```diff
@@ -0,0 +1,130 @@
+# Install self-hosted TimescaleDB on archlinux-based systems
+You can host TimescaleDB yourself, on your Arch Linux system. These
+instructions use the `pacman` package manager.
+
+<highlight type="warning">
+If you have already installed PostgreSQL using a method other than the `pacman`
+package manager, you could encounter errors following these instructions. It is
+safest to remove any existing PostgreSQL installations before you begin. If you
+want to keep your current PostgreSQL installation, do not install TimescaleDB
+using this method.
+[Install from source](/install/latest/self-hosted/installation-source/)
+instead.
+</highlight>
+
+<procedure>
+
+### Installing self-hosted TimescaleDB on archlinux-based systems
+1.  At the command prompt, as root, add the PostgreSQL third party repository
+    to get the latest PostgreSQL packages:
+    ```bash
+    pacman -Syu timescaledb
+    ```
+1.  Update the `postgresql.conf` file, to use TimescaleDB as PostgreSQL extension:
+    ```bash
+    echo "shared_preload_libraries = 'timescaledb'" >> /var/lib/postgres/data/postgresql.conf
+    ```
+1.  Install `timescaledb-tune`:
+    ```bash
+    pacman -Syu timescaledb-tune
+    ```
```

Comment:
```suggestion
1.  Install TimescaleDB and timescaledb-tune:
    ```bash
    pacman -Syu timescaledb timescaledb-tune
    ```
1.  Run timescaledb-tune to adjust your `postgresql.conf` file, to use TimescaleDB as PostgreSQL extension:
    ```bash
    timescaledb-tune
    ```
```
Since timescaledb is part of the official archlinux repo we do not need to add 3rd party repo.
Instead of manually editing postgresql.conf we should run timescaledb-tune here.